ui/src/app/app-routing.tsx

```diff
@@ -59,6 +59,11 @@ const registrationGuard: Guard = {
   redirectPath: '/'
 };
 
+const expiredGuard: Guard = {
+  allowed: (): boolean => !profileStore.get().profile.renewableAccessModules.anyModuleHasExpired,
```

Comment:
This is not the correct check.  A user who has never completed a module is not "expired" but they should also not be able to access the system normally.

Would you mind implementing this check as a function like `hasRegisteredAccess()`?  I would expect that we'll need this elsewhere.

Reply:
I'm not sure I am following. We can have multiple guards applied to a set of routes.
Since we have a registration guard, this guard's job is strictly to check if the user has an expired module.

In the case where they somehow have both an expired and incomplete module, they would first be redirected to AAR, once completing that, they would be redirected to complete their registration. It may be a bit awkward, but that seems like a corner case we can optimize for if necessary.

If what I am saying doesn't make sense, let's chat.

Reply:
OK - "no data at all" is not a real use case you need to account for here